easycv/io/input.py

```diff
@@ -68,6 +69,24 @@ def open_image(path):
         raise InvalidPathError("File path is invalid.") from None
 
 
+def random_dog_image():
+    """
+    Makes a request to `DogApi <https://dog.ceo/dog-api/>`_ for a random image and
+    extracts the link from the response.
+
+    :return: Link to a random dog image
+    :rtype: :class:`str`
+    """
+    response = urlopen("https://dog.ceo/api/breeds/image/random")
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Audit url open for permitted schemes. Allowing use of file:/ or custom schemes is often unexpected.](https://app.codacy.com/gh/easycv/easycv/pullRequest?prid=5649893)